app/controllers/api/base_controller/manager.rb

```diff
@@ -140,7 +140,7 @@ def update_multiple_collections(is_subcollection, target, type, resources)
           update_one_collection(is_subcollection, target, type, rid, r)
         end
         raise BadRequestError, "No #{type} resources were specified for the #{action} action" if processed == 0
-        {"results" => results}
+        {"results" => results.flatten}
```

Comment:
@abellotti it can't be placed there, because it calls `assign_tags_resource` for each resource. So even though that is flattened, it is still returning a single array, which results in the results being an array of an array